app/controllers/pet_applications_controller.rb

```diff
@@ -5,4 +5,17 @@ def index
     @applications = @pet.applications.all
   end
 
+  def update
+    pet = Pet.find(params[:id])
```

Comment:
Nice job figuring out pet.update to change a is_adoptable status. It was great you already had a helper method for the status change!